site/sass/components/_card.scss

```
@@ -37,16 +37,29 @@
     /* stylelint-enable no-descending-specificity */
   }
 
+  &__topic {
+    [dir='ltr'] & {
+      margin-right: 0.625rem;
+    }
+
+    [dir='rtl'] & {
+      margin-left: 0.625rem;
+    }
```

```suggestion
    margin-inline-end: 0.625rem;
    
    @supports not (margin-inline-end: 1rem) {
      [dir='ltr'] & {
        margin-right: 0.625rem;
      }

      [dir='rtl'] & {
        margin-left: 0.625rem;
      }
    }
```

This looks like a great opportunity to use [`margin-inline`](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline) where supported! Here, we use `@supports`, which is [super well supported](https://caniuse.com/css-featurequeries) to fall back to the higher-specificity direction based margins if inline margin isn't supported, but otherwise use the logical property.